desktop/core/src/desktop/js/apps/editor/EditorViewModel.js

```diff
@@ -91,6 +91,9 @@ export default class EditorViewModel {
     this.editorTitle = ko.pureComputed(
       () => this.activeConnector() && this.activeConnector().displayName
     );
+    this._editorTitle = ko.pureComputed(() =>
```

Comment:
Why the duplicate?

Also note that you'd have to add the same logic for editor v1.

Reply:
Done.